src/utils/options/icons.ts

```diff
@@ -0,0 +1,16 @@
+export const iconNameAllowed = [
```

Comment:
could you move this array to 'icons' folder? because util functions must have general purpose and also you can get same array in this way:
```
const iconNamesAllowed = Object.keys(Icons )

Reply:
great idea! I was making things to complicated! Done!